### ISSUE TITLE
Embedded content of an RTL note appears LTR in an LTR note

### ISSUE THREAD

**Issue by psyguy** [esm7/obsidian-rtl#10]
Hi there,

The problem arises when note `A.md` is LTR and `B.md` is RTL, and you want to embed `![[B.md]]` in `A.md`.

The embedded content follows the text direction of the host note, i.e., `A.md`.

Thanks.

**Comment by esm7** [comment]
Interesting, it might be fixable by selectively embedding `direction` tags in the markdown preview for each embed.

**Comment by lingdocs** [comment]
Yes, in fact adding a `direction="auto"` attribute to all the `<p>` tags etc. would be the best/easiest solution in my opinion. See [this piece of the MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/dir). With this simple tag, (`direction="auto") basically all RTL-LTR issues are solved. ([People would like to see paragraph by paragraph RTL-LTR switching](https://forum.obsidian.md/t/rtl-support/1438/54), and this one html attribute takes care of it. Unfortunately there's no `direction: auto` in CSS but the html attribute works fantastic.
